src/System.Runtime.InteropServices.RuntimeInformation/src/System.Runtime.InteropServices.RuntimeInformation.builds

```diff
@@ -15,13 +15,22 @@
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="System.Runtime.InteropServices.RuntimeInformation.csproj">
-      <TargetGroup>net46</TargetGroup>
+      <OSGroup>Windows_NT</OSGroup>
```

Comment:
Don't set this for desktop assemblies since those need to be built on Unix to be put in the reference package.